conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/EndpointRegistry.java

```diff
@@ -27,6 +27,6 @@
  */
 public interface EndpointRegistry {
 
-    EndpointRegistry add(Endpoint description, HttpHandler handler);
+    void add(Endpoint endpoint);
```

Comment:
Should we use `void register(Endpoint endpoint)` instead?

Reply:
👍